apps/console/src/features/organizations/components/organization-switch/organization-switch-dropdown.tsx

```diff
@@ -104,20 +104,38 @@ const OrganizationSwitchDropdown: FunctionComponent<OrganizationSwitchDropdownIn
         feature.organizations
     ]);
 
-    const getOrganizationList = useCallback((filter: string, after: string, before: string) => {
+    const getOrganizationList = useCallback(async (filter: string, after: string, before: string) => {
+        let superOrg: OrganizationInterface;
+
+        try {
+            const superOrgId: string = await getUserSuperOrganization();
+
+            if (currentOrganization.id !== superOrgId) {
+                superOrg = {
+                    id: superOrgId,
+                    name: "Super",
```

Comment:
User's root org name will not always be **Super** right? If so, I think we better change this to that org's name

Reply:
Since the API only returns the org id for the time being, the org name has not been used here.